packages/appkit/src/networks/index.ts

```diff
@@ -1,195 +1,66 @@
-  id: 'eip155:56',
-  chainId: 56,
-  name: 'Binance Smart Chain',
-  currency: 'BNB',
-  explorerUrl: 'https://bscscan.com',
-  rpcUrl: getBlockchainApiRpcUrl(56, 'eip155'),
-  chainNamespace: 'eip155'
-}
-
-export const optimism: CaipNetwork = {
-  id: 'eip155:10',
-  chainId: 10,
-  name: 'Optimism',
-  currency: 'ETH',
-  explorerUrl: 'https://optimistic.etherscan.io',
-  rpcUrl: getBlockchainApiRpcUrl(10, 'eip155'),
-  chainNamespace: 'eip155'
-}
-
-export const polygon: CaipNetwork = {
-  id: 'eip155:137',
-  chainId: 137,
-  name: 'Polygon',
-  currency: 'MATIC',
-  explorerUrl: 'https://polygonscan.com',
-  rpcUrl: getBlockchainApiRpcUrl(137, 'eip155'),
-  chainNamespace: 'eip155'
-}
-
-export const gnosis: CaipNetwork = {
-  id: 'eip155:100',
-  chainId: 100,
-  name: 'Gnosis',
-  currency: 'xDAI',
-  explorerUrl: 'https://gnosis.blockscout.com',
-  rpcUrl: getBlockchainApiRpcUrl(100, 'eip155'),
-  chainNamespace: 'eip155'
-}
-
-export const zkSync: CaipNetwork = {
-  id: 'eip155:324',
-  chainId: 324,
-  name: 'ZkSync',
-  currency: 'ETH',
-  explorerUrl: 'https://explorer.zksync.io',
-  rpcUrl: getBlockchainApiRpcUrl(324, 'eip155'),
-  chainNamespace: 'eip155'
-}
+export type { AppKitNetwork } from '@reown/appkit-common'
 
-export const zora: CaipNetwork = {
-  id: 'eip155:7777777',
-  chainId: 7777777,
-  name: 'Zora',
-  currency: 'ETH',
-  explorerUrl: 'https://explorer.zora.energy',
-  rpcUrl: getBlockchainApiRpcUrl(7777777, 'eip155'),
-  chainNamespace: 'eip155'
-}
-
-export const celo: CaipNetwork = {
-  id: 'eip155:42220',
-  chainId: 42220,
-  name: 'Celo',
-  currency: 'CELO',
-  explorerUrl: 'https://explorer.celo.org/mainnet',
-  rpcUrl: getBlockchainApiRpcUrl(42220, 'eip155'),
-  chainNamespace: 'eip155'
-}
-
-export const base: CaipNetwork = {
-  id: 'eip155:8453',
-  chainId: 8453,
-  name: 'Base',
-  currency: 'BASE',
-  explorerUrl: 'https://basescan.org',
-  rpcUrl: getBlockchainApiRpcUrl(8453, 'eip155'),
-  chainNamespace: 'eip155'
-}
-
-export const aurora: CaipNetwork = {
-  id: 'eip155:1313161554',
-  chainId: 1313161554,
-  name: 'Aurora',
-  currency: 'ETH',
-  explorerUrl: 'https://explorer.aurora.dev',
-  rpcUrl: getBlockchainApiRpcUrl(1313161554, 'eip155'),
-  chainNamespace: 'eip155'
-}
-
-export const sepolia: CaipNetwork = {
-  id: 'eip155:11155111',
-  chainId: 11155111,
-  name: 'Sepolia',
-  currency: 'ETH',
-  explorerUrl: 'https://sepolia.etherscan.io',
-  rpcUrl: getBlockchainApiRpcUrl(11155111, 'eip155'),
-  chainNamespace: 'eip155'
+export function getBlockchainApiRpcUrl(chainId: number | string, namespace: ChainNamespace) {
+  return `https://rpc.walletconnect.org/v1/?chainId=${namespace}:${chainId}`
 }
 
-export const baseSepolia: CaipNetwork = {
-  id: 'eip155:84532',
-  chainId: 84532,
-  name: 'Base Sepolia',
-  currency: 'BASE',
-  explorerUrl: 'https://sepolia.basescan.org',
-  rpcUrl: getBlockchainApiRpcUrl(84532, 'eip155'),
-  chainNamespace: 'eip155'
+export function defineChain<
```

Comment:
Maybe move this function to another file and keep this file only for network objects?

Reply:
I think it's fine here if not used elsewehere